examples/stackdriver-trace/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "example-stackdriver-trace",
+  "private": true,
+  "version": "0.3.1",
+  "description": "Example of using @opentelemetry/exporter-stackdriver-trace in Node.js",
+  "main": "index.js",
+  "scripts": {
+    "start": "cross-env GOOGLE_APPLICATION_CREDENTIALS=service_account_key.json node ./index.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/open-telemetry/opentelemetry-js.git"
+  },
+  "keywords": [
+    "opentelemetry",
+    "http",
+    "tracing"
+  ],
+  "engines": {
+    "node": ">=8"
+  },
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/open-telemetry/opentelemetry-js/issues"
+  },
+  "dependencies": {
+    "@google-cloud/storage": "^4.1.3",
```

Comment:
`@google-cloud/storage`  is unused

Reply:
It was in an earlier version. good catch